lib/ravelin/response.rb

```diff
@@ -36,8 +38,10 @@ def event(response_body)
     end
 
     def tag(response_body)
-      @tag_names    = response_body['tagNames']
-      @label        = response_body['label']
+      @tag_names              = response_body['tagNames']
+      @label                  = response_body['label'] if response_body.key?('label')
+      @tags                   = response_body['tags'] if response_body.key?('tags')
+      @client_reviewed_status = response_body['clientReviewedStatus'] if response_body.key?('clientReviewedStatus')
```

Comment:
what is this?

Reply:
It's what gets returned if the customer has already been tagged with a tag you're trying to add